menu/navigation.json

```diff
@@ -120,10 +120,6 @@
                     "label": "Join an organization",
                     "slug": "join-an-organization"
```

Comment:
This link gives a 404 when clicked from the side menu, the page seems to have been deleted so I think the link should be too. Check this is on the list of redirects.